sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusProcessorClient.java

```diff
@@ -169,8 +169,10 @@ public void onNext(ServiceBusMessageContext serviceBusMessageContext) {
                             processMessage.accept(serviceBusReceivedMessageContext);
```

Comment:
should accept be called if AutoComplete is disabled ?

Reply:
`accept` calls the user's message handler. It's not related to `autoComplete`